provider/boltmem/boltmem.go

```diff
@@ -30,38 +30,59 @@ import (
 var (
 	bktNotificationInfo = []byte("notification_info")
 	bktSilences         = []byte("silences")
-	bktAlerts           = []byte("alerts")
+	// bktAlerts           = []byte("alerts")
```

Comment:
Yea, I know it's not exactly best practice. 
But I want to rework functional tests at some point and am worried to forget about it.